notebook/tests/selenium/utils.py

```diff
@@ -256,14 +260,15 @@ def edit_cell(self, cell=None, index=0, content="", render=False):
         for line_no, line in enumerate(content.splitlines()):
             if line_no != 0:
                 self.current_cell.send_keys(Keys.ENTER, "\n")
+            self.current_cell.send_keys(Keys.ENTER, Keys.HOME)  # Avoid auto-indent.
```

Comment:
Before I discovered the `prefill_notebook` fixture, I was having trouble with auto-indentation when setting cell content to a multiline string that ended in a colon. This resets the cursor to the start of the line before sending the line content.

It is now redundant because the display_id test uses `prefill_notebook`, which doesn't have that problem, but I figured it's a useful fix regardless.

Reply:
Hm, looks like this doesn't quite work as I intended, so I'll remove it for now.